install-base.sh

```diff
@@ -196,11 +210,6 @@ fi
 echo "[[ -f /opt/rose/etc/rose-bash-completion ]] && . /opt/rose/etc/rose-bash-completion" >>/home/vagrant/.bashrc
 echo "[[ -f /opt/cylc/conf/cylc-bash-completion ]] && . /opt/cylc/conf/cylc-bash-completion" >>/home/vagrant/.bashrc
 
-#### Configure firefox as the default PDF viewer
-sudo -u $(logname) mkdir -p /home/vagrant/.local/share/applications
-sudo -u $(logname) bash -c 'echo "[Added Associations]" >/home/vagrant/.local/share/applications/mimeapps.list'
-sudo -u $(logname) bash -c 'echo "application/pdf=firefox.desktop;" >>/home/vagrant/.local/share/applications/mimeapps.list'
-
```

Comment:
I suspect this was only added to allow viewing of the old PDF versions of the documentation which we no longer produce.